package.json

```diff
@@ -25,11 +25,12 @@
     "dist": "script/dist",
     "build-storybook": "build-storybook -o build",
     "check-links": "script/check-links http://localhost:3000/css -v",
-    "lint": "stylelint --quiet --syntax scss src/**/*.scss",
+    "lint": "npm-run-all -s lint-css lint-js",
+    "lint-css": "stylelint --quiet --syntax scss src/**/*.scss",
     "lint-js": "eslint lib docs",
     "now-build": "next build",
     "now-start": "next start",
-    "postlint": "npm run lint-js",
+    "postversion": "git commit -m \"chore: v$npm_package_version\" package*.json",
```

Comment:
What's the benefit of doing this?

Reply:
This seems a little over-engineered to me. I think most contributors know how to manually change the version, and I think it makes the PR instructions actually more confusing

Reply:
I still think it's not totally necessary tbh. Most people already know how to upgrade a version manually and I'd rather stick to simple processes across all of our repos that are consistent.